### PR TITLE
Dropdown menu

### PR DESCRIPTION
**Pull request by Habtamu210**
Added JavaScript function for navigation bar

### CODE REVIEW

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# INVALID ❌ ❌ ❌ 

Hi @Habtamu210, 👋

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌
 - You added the eslintrc.json file correctly to your project but for the Github PR run the eslinter checks on your PR the linters.yml file in the .github/workflows/ folder needs to be updated too. Kindly update your linters.yml file following the instructions on [this page](https://github.com/microverseinc/linters-config/tree/master/html-css-js).

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

✅ Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_⚠️ Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

----
_Invalid Code Review Request does not count into the code reviews limit._

**Comment by prolajumokeoni** [comment]
Hi @Habtamu210 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have linters setup properly, This is the first project in which you should set up both [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters in the project repository.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Abenezer-Tilahun — COMMENTED** [review]
- [ ] Hello @Habtamu210 kindly check the comment

**Comment by Habtamu210** [comment]
@abenezer-tlahun.
How are you doing Abenezer? I can not fix this problem

(C:\Users\hp\Documents\portfoliopage\node_modules\stylelint\lib\utils\configurationError.js:10:14)

    at
C:\Users\hp\Documents\portfoliopage\node_modules\stylelint\lib\getConfigForFile.js:56:11
    at async Promise.all (index 0)

File: .stylelintrc.json [hunk 1]
```diff
@@ -1,20 +0,0 @@
...
-    "scss/at-rule-no-unknown":  [
-      true,
-      {
-        "ignoreAtRules": ["tailwind", "apply", "variants", "responsive", "screen"]
-      }
-    ],
-    "csstree/validator": true
-  },
-  "ignoreFiles": ["build/**", "dist/**", "**/reset*.css", "**/bootstrap*.css", "**/*.js", "**/*.jsx"]
-}
```
Comment (Abenezer-Tilahun) [inline .stylelintrc.json]: - [ ] Hello @Habtamu210 the reason is you removed configuration of .stylelintrc.json, Kindly add it again https://github.com/microverseinc/linters-config/blob/master/javascript/.stylelintrc.json

**Review by Abenezer-Tilahun — COMMENTED** [review]
Hi @Habtamu210

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
Invalid Code Review Request

### Invalid Code 
You have submitted a project, that does not pass stylenit linters setup properly, Kindly check linters are green before the submit for review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @abenezer-tlahun.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: script.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+});
+
+document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click', () => {
+  hamburger.classList.remove('active');
+  navMenu.classList.remove('active');
+}));
```
Comment (Whoistolu) [inline script.js]: - [ ] [REQUIRED] Kindly ensure that your design looks exactly like the design provided in template 2 of the figma. Let's stick to the properties e.g. font, colors, text, margins.
 ### Your Design
Screenshot 2022-11-17 at 00 15 25

### Figma Design
Screenshot 2022-11-17 at 00 17 33

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @Habtamu210 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._
- [ ] [REQUIRED] I am recommending these changes here because we do not have it in the file changes. Kindly ensure that the About menu option takes a user to the About page. Presently, you have routed it to the Work page. Please fix this.
https://github.com/Habtamu210/portfoliopage/blob/89e31bab48a628bbf8bdf782f3d95a5d6672abac/index.html#L25

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+const hamburger = document.querySelector('.hamburger');
+const navMenu = document.querySelector('.nav-menu');
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+});
+document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click', () => {
```
Comment (Davidosky007) [inline script.js:7]: * As mentioned by the previous reviewer your drop-down menu is still different from the given design Kindly work on it 🙏 
## Yours 
image
## Figma design 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Habtamu210 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
* No linter errors 👍 
* Professional looking PR 👍 
* Proper use of GitHub flow pattern 👍  

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidosky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @Habtamu210, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**